arbitrator/prover/src/machine.rs

```diff
@@ -531,6 +542,36 @@ impl Module {
             memory_store_internal_type,
         ));
 
+        //we're adding two internal functions here which implement in WAVM the
+        //memory.fill and memory.copy instructions in WASM
```

Comment:
```
// <text>
// <text>
```
Here and elsewhere

Reply:
Doesn't seem like `cargo fmt` complains about this, but happy to do it.

Reply:
`cargo fmt` can't tell they aren't build directives, which don't get a space. In our codebase
```
//action
```
means something is happening in excess of it being a comment